packages/angular/migrations.json

```diff
@@ -1836,6 +1836,22 @@
           "alwaysAddToPackageJson": false
         }
       }
+    },
+    "15.8.0-jest": {
+      "version": "15.8.0-beta.0",
+      "requires": {
+        "@angular-devkit/build-angular": ">=13.0.0 <16.0.0",
+        "@angular/compiler-cli": ">=13.0.0 <16.0.0",
+        "@angular/core": ">=13.0.0 <16.0.0",
+        "@angular/platform-browser-dynamic": ">=13.0.0 <16.0.0",
+        "jest": "^29.0.0"
```

Comment:
Perhaps @leosvelperez can confirm, but i feel like this should potentially be `>=29.0.0 < 30.0.0`

Reply:
that's the same thing. ^ is any minor of v29 -> v29.x.x. so >v29.0.0 < v30.0.0 -> v29.x.x

IIRC the list should match what the peers of the library require and this is what jest-preset-angular has (we took TS out since TS isn't really needed as Nx controls that and it's always going to be higher than that dep)

![WkMacM1-VUl87AtA_02231242](https://user-images.githubusercontent.com/23272162/221001024-77320cea-3105-412c-b272-feedc17b665d.png)